scripts/generate_reports.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+for var in "$@"
+do
+    	case $var in
+		--help)
```

Comment:
Though you may want to add --usage) just below, so both would do the same thing, or make it default, in case I run your script with --this-is-a-weird-parameter-and-what-now?